demo/admin/src/products/future/generated/ProductPriceForm.tsx

```diff
@@ -94,6 +95,11 @@ export function ProductPriceForm({ id }: FormProps): React.ReactElement {
                         component={FinalFormInput}
                         type="number"
                         label={<FormattedMessage id="product.price" defaultMessage="Price" />}
+                        startAdornment={
+                            <InputAdornment position="start">
+                                <FormattedMessage id="price.startAdornment" defaultMessage="€" />
```

Comment:
I don't think a string adornment needs translation, usually it will be used for units